jobs/dd-agent/templates/config/datadog.yaml.erb

```diff
@@ -89,6 +89,9 @@ skip_ssl_validation: <%= p("dd.skip_ssl_validation", "no") %>
     if p("dd.unique_friendly_hostname", false)
       hostname = "#{hostname}-#{spec.deployment}"
     end
+    if p("dd.friendly_hostname_append_vm_uuid", false)
+      hostname = "#{hostname}-#{spec.id}"
```

Comment:
Will this adhere to the hostname RFC convention?

Reply:
Yes, spec.id is only alphanumeric and dashes